front/src/common/Pathfinding/pathfinding.scss

```diff
@@ -0,0 +1,70 @@
+.pathfinding-main-container {
```

Comment:
why not put scss file in /styles/common ? And create a common.scss file who add each file inside /styles/common/ ?

Reply:
why not :)